crime-means-assessment/src/test/java/uk/gov/justice/laa/crime/meansassessment/service/FullAssessmentAvailabilityServiceTest.java

```diff
@@ -28,25 +29,28 @@ public void setup() {
         fullAssessmentAvailabilityService = new FullAssessmentAvailabilityService();
         meansAssessmentRequest = TestModelDataBuilder.getCreateMeansAssessmentRequest(true);
         meansAssessmentResponse = TestModelDataBuilder.getCreateMeansAssessmentResponse(true);
+        meansAssessmentRequest.setFullAssessmentDate(null);
     }
 
     @Test
-    public void whenAssessmentDateIsNotNull_thenFullAssessmentAvailableIsTrue() {
+    public void givenMeansAssessmentRequestAndResponse_WhenFullAssessmentDateIsNotNull_ThenFullAssessmentAvailableIsTrue() {
+        meansAssessmentResponse.setResult("");
```

Comment:
Not sure it's necessary to set the result here

Reply:
I would leave this as it covers the empty check for enum